### PR TITLE
MON-3694: chore: encourage the use of the new 'slices' package instead of 'golang.org/x/exp/slices'

### PR DESCRIPTION
**Pull request by machine424**
…ng.org/x/exp/slices'

* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by machine424** [comment]
/retitle MON-3694: chore: encourage the use of the new 'slices' package instead of 'golang.org/x/exp/slices'

File: .golangci.yaml [hunk 1]
```diff
@@ -36,7 +36,10 @@ linters-settings:
       main:
         deny:
           - pkg: "github.com/pkg/errors"
-            desc: "Use 'errors' or 'fmt' instead of github.com/pkg/errors"
+            desc: "Use 'errors' or 'fmt' instead"
+          # Maybe we'll need that back for some new exp stuff.
```
[the remark this answers is not in the document]
Comment (machine424) [reply .golangci.yaml]: Just to diminish doubts and prevent overthinking :)

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by danielmellado** [comment]
@bburt-rh mind checking the docs-approved label, please? Thanks!

**Comment by danielmellado** [comment]
@machine424 please lif the hold label whenever you're fine with it. We'll still be needing some labels tho

**Comment by machine424** [comment]
It was held by the bot again
/hold cancel
This should get it merged.
I think `acknowledge-critical-fixes-only` is no longer required.

**Pull request merged**